build.cake

```diff
@@ -454,7 +454,7 @@ Task("Default")
 .IsDependentOn("UploadCoverage")
 .IsDependentOn("AssertPackages")
 .IsDependentOn("NuGet")
-.IsDependentOn("GitHubRelease")
+//.IsDependentOn("GitHubRelease")
```

Comment:
Please remove the import in line 1 aswell (we often get problems there too) and the complete task. Please remove comment

Reply:
completely removed